src/cloudant/client.py

```diff
@@ -210,27 +210,54 @@ def delete_database(self, dbname):
         if dbname in list(self.keys()):
             super(CouchDB, self).__delitem__(dbname)
 
-    def db_updates(self, since=None, continuous=True):
-        """
-        Streams data from _db_updates feed. Yields information about
-        databases that have been updated.
-
-        :param str since: Update streaming starts from this sequence identifier.
-        :param bool continuous: Dictates the streaming of data.
-            Defaults to True.
-
-        :returns: Iterable stream of database updates
-        """
-        db_updates_feed = Feed(
-            self.r_session,
-            posixpath.join(self.cloudant_url, '_db_updates'),
-            since=since,
-            continuous=continuous
-        )
-
-        for update in db_updates_feed:
-            if update:
-                yield update
+    def db_updates(self, raw_data=False, **kwargs):
+        """
+        Returns the ``_db_updates`` feed iterator.  The ``_db_updates`` feed can
+        be iterated over and once complete can also provide the last sequence
+        identifier of the feed.
+
+        For example:
+
+        .. code-block:: python
+
+            # Iterate over a "normal" _db_updates feed
+            db_updates = db.db_updates()
+            for db_update in db_updates:
+                print(db_update)
+            print(db_updates.last_seq)
+
+            # Iterate over a "continuous" _db_updates feed with additional options
+            db_updates = db.db_updates(feed='continuous', since='now', descending=True)
+            for db_update in db_updates:
+                print(db_update)
+
+        :param bool raw_data: Dictates whether the data returned by the feed is
```

Comment:
Fixed in both client and database.  Done in d28550d22672b0558ed7dfafe75839b299bc3484.